scene/2d/cpu_particles_2d.cpp

```diff
@@ -104,7 +104,8 @@ void CPUParticles2D::set_use_local_coordinates(bool p_enable) {
 
 	// We only need NOTIFICATION_TRANSFORM_CHANGED
 	// when following an interpolated target.
-	set_notify_transform(_interpolation_data.interpolated_follow);
+	set_notify_transform(_interpolation_data.interpolated_follow || !p_enable);
```

Comment:
The new condition part is relevant only for editor.

Reply:
Sorry @KoBeWi but what would be the best way to structure this? Would it be ok to do the below? That would result in `set_notify_transform()` getting called twice when in editor mode, but it would strip out nicely for runtime. Is there a better way to do it.
```cpp
	set_notify_transform(_interpolation_data.interpolated_follow);

#ifdef TOOLS_ENABLED
	if (Engine::get_singleton()->is_editor_hint()) {
		set_notify_transform(_interpolation_data.interpolated_follow || !local_coords);
	}
#endif
```

Reply:
```C++
#ifdef TOOLS_ENABLED
	set_notify_transform(_interpolation_data.interpolated_follow || !p_enable);
#else
	set_notify_transform(_interpolation_data.interpolated_follow);
#endif

Reply:
Ah I'm dumb. Thanks! Done.